packages/webapp/src/components/Task/PureTaskTypeSelection/PureTaskTypeSelection.jsx

```diff
@@ -123,7 +126,15 @@ export const PureTaskTypeSelection = ({
           {taskTypes
             ?.filter(({ farm_id, task_translation_key }) => {
               const supportedTaskTypes = getSupportedTaskTypesSet(isAdmin);
-              return farm_id === null && supportedTaskTypes.has(task_translation_key);
+              // If trying to make a task through the crop management plan Add Task link -- exclude animal tasks from selection for now
+              const isNotAnimalTaskWhileCreatingCropTask = !(
```

Comment:
Not a blocker for merging, but the variable name is confusing me more 😂 I'd rename it to `shouldDisplayTaskType`, which is the result, and then the statement afterwards is what contains the logic for displaying or not

Reply:
Oh, I specifically did not group it with the other logic... I personally prefer to separate long tough to read logic into "readable chunks" . So in my head I read the return statement below as `isLiteFarmDefaultTaskType && isSupportedTaskType && isNotAnimalTaskWhileCreatingCropTask`.

The name is wordy I am open to a better name ahaha. But the logic points do seem distinct to me. 

I did an inbetween change to address this comment -- let me know if its ok!